src/components/SummaryBox/SummaryBox.stories.tsx

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import React from 'react'
+import { SummaryBox } from './SummaryBox'
+
+export default {
+  title: 'Components/SummaryBox',
```

Comment:
Nit: can we name this `Components / Summary box` so it matches the [USWDS name](https://designsystem.digital.gov/components/summary-box/)?